test/e2e-olm/utils.go

```diff
@@ -35,27 +37,47 @@ const (
 	exampleSecretsPath   = "/tmp/example-secrets"
 )
 
-func registerSchemes(t *testing.T) {
-	elasticsearchList := &loggingv1.ElasticsearchList{}
-	err := test.AddToFrameworkScheme(loggingv1.SchemeBuilder.AddToScheme, elasticsearchList)
+func setupK8sClient(t *testing.T) {
+	if _, ok := os.LookupEnv(TestOperatorNamespaceEnv); !ok {
+		t.Fatal("TEST_OPERATOR_NAMESPACE is unset")
+	}
+	operatorNamespace = os.Getenv(TestOperatorNamespaceEnv)
+	t.Logf("Found namespace: %q", operatorNamespace)
+
+	projectRootDir = getProjectRootPath("elasticsearch-operator")
+
+	var err error
+	cfg, err := config.GetConfig()
 	if err != nil {
-		t.Fatalf("failed to add custom resource scheme to framework: %v", err)
+		t.Fatalf("Error get config: %s", err)
+	}
+	if cfg == nil {
+		t.Fatal("config is nil")
+	}
+
+	registerSchemes(t)
+
+	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
+	if err != nil {
+		t.Fatalf("Error get k8sClient: %s", err)
+	}
+	if k8sClient == nil {
+		t.Fatal("k8sClient is nil")
 	}
```

Comment:
Suggest to add this check in the first if statement, e.g.:
```
if err != nil || cfg == nil {
}
```

Reply:
Is this necessary since cfg == nil is checked at line 54?